src/modules/reading-room/const/query-params.ts

```diff
@@ -0,0 +1,30 @@
+import { QueryParamConfig } from 'use-query-params';
+
+import { AdvancedFilter } from '@reading-room/types';
+
+const divider = '--';
```

Comment:
will this work correctly if someone searches for:
name should contain `huis--van-allijn`
does it split on first occurrence or on every occurrence?

Reply:
Fixed in https://github.com/viaacode/hetarchief-client/pull/198/commits/50c15e6990fe72946cc1a782f7c9883ab888f1f5

http://localhost:3200/leeszaal/leeszaal-8?advanced=afaahuis--van-alijn%2Cafaa%252Cbanaan%252C%2520perzik%2Cafabmango